Functions/Engines/Wine/script.js

```diff
@@ -177,6 +177,16 @@ Wine.prototype.run = function (executable, args, captureOutput) {
         args = [];
     }
 
+    var extensionFile = executable.split(".").pop();
+
+    if(extensionFile == "msi") {
+        return this.run("msiexec", ["/i", executable], captureOutput);
```

Comment:
As I said in another comment already, I'm not sure this is really a good idea. You loose completely the ability to run a msi with anything else than `/i`.